doc/user/bgp.rst

```diff
@@ -1341,6 +1343,49 @@ Configuring Peers
 
    This command is only allowed for eBGP peers.
 
+.. index:: [no] neighbor <A.B.C.D|X:X::X:X|WORD> as-override
+.. clicmd:: [no] neighbor <A.B.C.D|X:X::X:X|WORD> as-override
+
+   Override AS number of the originating router with the local AS number.
+
+   Usually this configuration is used in PEs (Provider Edge) to replace
+   the incoming customer AS number so the connected CE (Customer Edge)
+   can use the same AS number as the other customer sites. This allows
+   customers of the provider network to use the same AS number across
+   their sites.
+
+   This command is only allowed for eBGP peers.
+
+.. index:: [no] neighbor <A.B.C.D|X:X::X:X|WORD> allowas-in [<(1-10)|origin>]
+.. clicmd:: [no] neighbor <A.B.C.D|X:X::X:X|WORD> allowas-in [<(1-10)|origin>]
+
+   Accept incoming routes with AS path containing AS number with the same value
+   as the current system AS.
+
+   This is used when you want to use the same AS number in your sites, but you
+   can't connect them directly. This is an alternative to
+   `neighbor WORD as-override`.
+
+   The parameter `(1-10)` configures the amount of accepted occurences of the
+   system AS number in AS path.
+
+   The parameter `origin` configures BGP to only accept routes originated with
+   the same AS number as the system.
+
+   This command is only allowed for eBGP peers.
+
+.. index:: [no] neighbor <A.B.C.D|X:X::X:X|WORD> addpath-tx-all-paths
+.. clicmd:: [no] neighbor <A.B.C.D|X:X::X:X|WORD> addpath-tx-all-paths
+
+   Configure BGP to send all known paths to neighbor in order to preserve multi
+   path capabilities inside a network.
+
+.. index:: [no] neighbor <A.B.C.D|X:X::X:X|WORD> addpath-tx-bestpath-per-AS
+.. clicmd:: [no] neighbor <A.B.C.D|X:X::X:X|WORD> addpath-tx-bestpath-per-AS
+
+   Configure BGP to send best known paths to neighbor in order to preserve multi
```

Comment:
I think we should distinguish this command from `addpath-tx-all-paths`. I mean in the description they both say the same. Adding an additional description that it announces only the best paths per AS would be handy, IMO.

Reply:
@ton31337 
> I mean in the description they both say the same.

No, they don't! the first one says (watch the bold words):

> Configure BGP to send **all** known paths to neighbor in order to preserve multi
> path capabilities inside a network.

The one you commented says:

> Configure BGP to send **best** known paths to neighbor in order to preserve multi
> path capabilities inside a network.

> Adding an additional description that it announces only the best paths per AS would be handy, IMO.

I agree... Would you mind helping me find better wording please?

Reply:
OMG, kill me :smile: I read both sentenses kinda _automatically_ and missed that. Sorry.